src/Standards/PSR12/Docs/Classes/OpeningBraceSpaceStandard.xml

```diff
@@ -0,0 +1,32 @@
+<documentation title="Opening Brace">
+    <standard>
+    <![CDATA[
+    Opening braces must not be followed by a blank line.
```

Comment:
Same remark about the description as I left on #170 - I think the context (OO structures only) needs to be made clear.